docs/reverse.rst

```diff
@@ -221,6 +221,78 @@ Alternatively, you can define SANs in the Traefik static configuration using rou
 
 .. _`Traefik`: https://traefik.io/
 
+Linuxserver Swag reverse proxy
+------------------------------
+
+Another popular reverse proxy is Linuxserver Swag with built-in ``letsencrypt``. It can be placed in front of the Mailu front container and used with other docker-containers in ``docker-compose.yml``:
```

Comment:
The ACME client hasn't been called `letsencrypt` for *years*: [it's](https://github.com/linuxserver/docker-swag/blob/5df55cc0c0e207ab9833b61fc1ce05a9b5f9058d/Dockerfile#L101) [certbot](github.com/certbot/certbot)

Reply:
> The ACME client hasn't been called `letsencrypt` for _years_: [it's](https://github.com/linuxserver/docker-swag/blob/5df55cc0c0e207ab9833b61fc1ce05a9b5f9058d/Dockerfile#L101) [certbot](https://github.com/certbot/certbot)

The [letsencrypt](https://github.com/Mailu/Mailu/search?q=certbot) name is still used in file names and code.

Reply:
Fixed in 779bb27dee5ea1f351775bef781a1962bffc70b6, based on [Swag doc source](https://docs.linuxserver.io/general/swag#cert-provider-lets-encrypt-vs-zerossl)